adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -172,6 +172,7 @@ type CorsConfig struct {
 	AccessControlAllowMethods     []string `mapstructure:"accessControlAllowMethods"`
 	AccessControlAllowOrigins     []string `mapstructure:"accessControlAllowOrigins"`
 	AccessControlExposeHeaders    []string `mapstructure:"accessControlExposeHeaders"`
+	Override                      bool     `mapstructure:"override"`
```

Comment:
We are not using this override flag yet right?

Reply:
The override property is now being used.